lib/ChatBotContainer.jsx

```diff
@@ -1,10 +1,10 @@
 import styled from 'styled-components';
 
 const ChatBotContainer = styled.div`
-  background: #f5f8fb;
+  background: ${props => (props.theme && props.theme.background) || '#f5f8fb'};
```

Comment:
Defining a default theme should be really easy:
for example, instead of:
`background: ${props => (props.theme && props.theme.background) || '#f5f8fb'};`
it could be like so:
`background: ${props => (props.theme && props.theme.background) || theme.default.background};`
and `theme` object should be defined somewhere and imported:
```
const theme = {
  default: {
    background: "#f5f8fb",
    color: "#fff",
  }
};
```